api.js

```diff
@@ -151,7 +151,7 @@ Api.prototype._run = function (files, _options) {
 					// continue to run.
 					runStatus.handleExceptions({
 						exception: err,
-						file: file
+						file: path.relative('.', file)
```

Comment:
This is more consistent with existing behavior in `fork`, etc. But maybe we should be emitting everything as absolute paths and have the reporters handle truncating to relative ones? Seems like it would be best to keep it explicit until we decide to show it to a human.